plugin/plugins/bilibili_danmaku/__init__.py

```
@@ -1095,6 +1095,28 @@ async def get_status(self, **_):
+    @plugin_entry(
+        id="open_ui",
+        name="打开弹幕控制台",
+        description="在浏览器中打开B站弹幕插件的Web UI控制台，用于配置直播间、查看弹幕等",
+        kind="action"
+    )
+    async def open_ui(self, **_):
+        """在浏览器中打开B站弹幕控制台"""
+        url = "http://localhost:48916/plugin/bilibili-danmaku/ui/"
+        try:
+            if sys.platform == "win32":
+                subprocess.Popen(["cmd", "/c", "start", "", url], shell=False)
```

Ruff (0.15.10)

[error] Undefined name `subprocess`

(F821)